--- src/main/http_main.c	2006-07-12 04:16:05.000000000 -0400
+++ src/main/http_main_new.c	2010-05-10 17:59:38.310235592 -0400
@@ -2981,9 +2981,6 @@
 #define NumSIG 32   /* for 1998's unixes, this is still a good assumption */
 #endif
 
-#ifdef SYS_SIGLIST /* platform has sys_siglist[] */
-#define INIT_SIGLIST()  /*nothing*/
-#else /* platform has no sys_siglist[], define our own */
 #define SYS_SIGLIST ap_sys_siglist
 #define INIT_SIGLIST() siglist_init();
 
@@ -3100,7 +3097,6 @@
         if (ap_sys_siglist[sig] == NULL)
             ap_sys_siglist[sig] = "";
 }
-#endif /* platform has sys_siglist[] */
 
 #ifdef AP_ENABLE_EXCEPTION_HOOK
 typedef struct except_hook_t {
--- src/main/Makefile.old	2010-05-10 18:20:05.299208185 -0400
+++ src/main/Makefile	2010-05-10 18:20:10.678276336 -0400
@@ -71,14 +71,14 @@
 uri_delims.h: gen_uri_delims
 	./gen_uri_delims >uri_delims.h
 
-gen_uri_delims: gen_uri_delims.o
-	$(CC) $(CFLAGS) $(LDFLAGS) -o gen_uri_delims gen_uri_delims.o $(LIBS)
+gen_uri_delims: gen_uri_delims.c
+	gcc -I../os/unix -I../include $(CFLAGS) $(LDFLAGS) -o gen_uri_delims gen_uri_delims.c $(LIBS)
 
 test_char.h: gen_test_char
 	./gen_test_char >test_char.h
 
-gen_test_char: gen_test_char.o
-	$(CC) $(CFLAGS) $(LDFLAGS) -o gen_test_char gen_test_char.o $(LIBS)
+gen_test_char: gen_test_char.c
+	gcc -I../os/unix -I../include $(CFLAGS) $(LDFLAGS) -o gen_test_char gen_test_char.c $(LIBS)
 
 # We really don't expect end users to use this rule.  It works only with
 # gcc, and rebuilds Makefile.tmpl.  You have to re-run Configure after
--- Makefile.old	2010-05-10 18:08:55.654343551 -0400
+++ Makefile	2010-05-10 18:09:13.652246313 -0400
@@ -409,7 +409,6 @@
 		$(CP) $(TOP)/$(SRC)/$${osdir}/os.h $(root)$(includedir)/; \
 		echo "$(CP) $(TOP)/$(SRC)/$${osdir}/os-inline.c $(root)$(includedir)/"; \
 		$(CP) $(TOP)/$(SRC)/$${osdir}/os-inline.c $(root)$(includedir)/
-	chmod 644 $(root)$(includedir)/*.h $(root)$(includedir)/xml/*.h
 	@echo "<=== [include]"
 
 #   create an initial document root containing the Apache manual,
