--- btpd-0.15/btpd/btpd.c	2009-02-09 14:12:28.000000000 -0500
+++ btpd-0.15-1/btpd/btpd.c	2010-05-15 07:24:31.460165860 -0400
@@ -113,7 +113,7 @@
         gethostname(idcon + n, sizeof(idcon) - n);
     idcon[sizeof(idcon) - 1] = '\0';
     n = strlen(idcon);
-
+    bzero (m_peer_id, sizeof(m_peer_id));
     SHA1(idcon, n, m_peer_id);
     bcopy(m_peer_id, &seed, sizeof(seed));
     bcopy(BTPD_VERSION, m_peer_id, sizeof(BTPD_VERSION) - 1);
